subsys/bluetooth/mesh/rpl.c

```diff
@@ -277,20 +283,16 @@ static void clear_rpl(struct bt_mesh_rpl *rpl)
 	}
 
 	(void)memset(rpl, 0, sizeof(*rpl));
+	atomic_clear_bit(store, rpl - &replay_list[0]);
```

Comment:
Maybe create a RPL_INDEX(rpl) macro to make it abundantly clear what this subtraction does?